docs/hydra/guides/openid.md

```diff
@@ -18,49 +19,43 @@ ory patch oauth2-config {project.id} \
 
 ## Pairwise algorithm
 
-The `pairwise` algorithm provides a different sub value to each client, ensuring that the clients cannot correlate the user's
-activities without permission. To use the `pairwise` algorithm, you must enable it and set
-`/oidc/subject_identifiers/pairwise/salt`. This salt is used to obfuscate the `sub` value.
+The `pairwise` algorithm provides a different subvalue to each client, ensuring that clients can't correlate user activity without
+permission. To use the `pairwise` algorithm, you must enable it and set `/oidc/subject_identifiers/pairwise/salt`. The salt is
+used to obfuscate the `sub` value.
 
-To enable both algorithms, use the following configuration:
+To enable the `pairwise` algorithm, run this command:
 
 ```shell
 ory patch oauth2-config {project.id} \
-  --replace "/oidc/subject_identifiers/supported_types=[\"public\", \"pairwise\"]" \
+  --replace "/oidc/subject_identifiers/supported_types=[\"pairwise\"]" \
   --replace "/oidc/subject_identifiers/pairwise/salt=\"{16-character-long-salt}\""
 ```
 
 :::warning
 
-Do not change the salt value once it is set in production. Changing it will cause all client applications to receive new user IDs
-from Ory, which can lead to serious complications with authentication.
+Don't change the salt value once it's set in production. When you change the salt, all client applications receive new user IDs
+from Ory. This can cause serious complications with authentication in your system.
 
 :::
 
-Each OAuth 2.0 client has a configuration field called `subject_type`. The value of that `subject_type` is either `public` or
-`pairwise`. If the identifier algorithm is enabled, Ory will automatically choose the right strategy.
+Each OAuth 2.0 client has a `subject_type` configuration field that can take a `public` or `pairwise` value. If the identifier
+algorithm is enabled, Ory automatically chooses the right strategy.
 
-While Ory handles `sub` obfuscation out of the box, you can also override this value with your own obfuscated sub value by setting
-`force_subject_identifier` when accepting the login request in your custom OAuth2 login app.
+While Ory handles `sub` obfuscation out of the box, you can also override this value with your own obfuscated `sub` value by
+setting `force_subject_identifier` when accepting the login request in your custom OAuth2 login app.
 
-## Using `login_hint` with a different subject
+## Using 'login_hint' with a different subject
 
-If a user has already logged in with a subject (for example, user-A), and she wants to log in as another user using `login_hint`
-(for example, login_hint=user-B), directly accepting the latter login request in your login provider will cause an error:
+If a user has already logged in with a subject (for example, "user-A"), and she wants to log in as another user using `login_hint`
```

Comment:
Did you mean to put user-A in back ticks (as in `user-A`), rather than double quotes?

Reply:
I used double quotes bc that is a sample user name that's not used in any command or URI fragment.